Dockerfile

```diff
@@ -1,15 +1,10 @@
 FROM node:erbium-alpine
 
-RUN npm install -g openhim-mediator-shell-script
+COPY . .
+
+RUN yarn
 
 # Create folder for the shell scripts
 RUN mkdir /opt/openhim-shell-scripts
```

Comment:
maybe we should change the base directory to be `/app` as this will become our working directory